launch.sh

```diff
@@ -31,10 +31,16 @@ while getopts "hd" option; do
    esac
 done
 
-# On versions above 20.10.2, docker-compose is docker compose
-smaller=$(printf "$(docker --version | egrep -o '[0-9]+\.[0-9]+\.[0-9]+')\n20.10.2" | sort -V | head -n1)
-if [[ "$smaller" == "20.10.2" ]]; then
-  docker compose up $options --remove-orphans --build
+# On versions above 20.10.2, docker-compose is "docker compose".
+DOCKER_STR=$(docker --version)
+DOCKER_VER=$(echo "$DOCKER_STR" | sed -n 's/.*\([0-9]\{2\}\.[0-9]\{2\}\.[0-9]\{2\}\).*/\1/p')
+
+echo "$DOCKER_VER ge 20.10.2"
+if dpkg --compare-versions $DOCKER_VER gt 20.10.2; then
```

Comment:
Can we get similar functionality without introducing the dependency on `dpkg`?

Reply:
Good question. Right. As I described in  the commit body of this PR,  because the Bash language does not deal with the floating point number. Fortunately, DPKG is the appropriate way because most modern Linux distributions support the "DPKG" package.

Reply:
We also have users on Windows and macOs, so I was thinking maybe we can revert to the old logic which was something like:

```
docker compose up -d --remove-orphans || docker-compose up -d --remove-orphans
```

This quite an ugly approach but it does so without any extra dependencies. 

WDYT?

Reply:
> We also have users on Windows and macOs, so I was thinking maybe we can revert to the old logic which was something like:
> 
> ```
> docker compose up $options --remove-orphans || docker-compose up $options --remove-orphans
> ```

@fdegier , Sorry, I missed this comment that you noted. 
I **agree** that we revert to the old logic more that current logic.  
Let's try to use the old logic as follows

`docker compose up $options --remove-orphans || docker-compose up $options --remove-orphans`